addons/mail/static/tests/translation/translation_test.js

```diff
@@ -0,0 +1,50 @@
+/* @odoo-module */
+
+import { click, start, startServer, contains } from "@mail/../tests/helpers/test_utils";
+
+const spanishText = "Al mal tiempo, buena cara.";
+const englishText = "To bad weather, good face.";
+const rpc = { value: englishText, source: "Spanish", error: null };
```

Comment:
This const is also useless, return value immediately in the test where it's used

Reply:
Removed.